static/admin/config.yml

```diff
@@ -38,6 +38,7 @@ collections:
       - {label: "Body", name: "body", widget: "markdown"}
       - {label: "Tags", name: "tags", widget: "list"}
       - {label: "isControl", name: "iscontrol", widget: "boolean", default: true}
+      - {label: "Version No.", name: "version", widget: "select", options: ["1", "2", "3","4", "5"]}
```

Comment:
Could we have versions like “1.01”, “1.02”,etc.. 
it will demo more like what they use today.

Reply:
updated